File: 15_july/MissingNumber.java
public class MissingNumber {
    public static void main(String[] args) {
        int[] arr = {1, 0, 2, 4};
        int n = 4;
        int total = n * (n + 1) / 2;

        for (int i : arr) {
            total = total - i;
        }

        System.out.println(total); //missing number is 3
    }
}

File: 15_july/MissingNumber.js
const arr = [1, 0, 2, 4];
const n = 4;
let total = n * (n + 1) / 2;

for (let i = 0; i < arr.length; i++) {
  total = total - arr[i];
}

console.log(total); //missing number=3
